app/views/merchants/dashboard.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Merchant: <%= @merchant.name %></h1>
```

Comment:
I think a lot of this dashboard can go into a partial, since it's going to be reused between pages